src/docs/reference/modules/Elasticsearch/docker-compose.yml

```diff
@@ -10,6 +10,10 @@ services:
       - cluster.initial_master_nodes=es01,es02
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      # Disable security features
+      - xpack.security.enabled=false
+      - xpack.security.http.ssl.enabled=false
+      - xpack.security.transport.ssl.enabled=false
```

Comment:
Why disabling security?

Reply:
Having to configure security is new in Elasticsearch 8. I figured that locally, for development, you don't need that in favor of an easier config.